client/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
```

Comment:
Unfortunately, the tab width set within the project was changed to 2 tabs, which is the cause of most of the conflicts in this merge 😭